flocker/node/script.py

```diff
@@ -294,7 +365,7 @@ class AgentServiceFactory(PClass):
     # This should have an explicit interface:
     # https://clusterhq.atlassian.net/browse/FLOC-1929
     deployer_factory = field(mandatory=True)
-    get_external_ip = field(initial=_get_external_ip, mandatory=True)
+    get_external_ip = field(initial=(lambda: _get_external_ip), mandatory=True)
```

Comment:
@moshez: Ah, I was misusing the term "vendored". But the point I was really trying to make is that the fixes here and below are all required for our tests to pass with the latest version (or our fork) of Pyrsistent.